tests/test_round_trip.rs

```diff
@@ -50,6 +55,8 @@ fn filter(entry: &DirEntry) -> bool {
         "tests/rust/src/test/run-pass/inner-attrs-on-impl.rs" |
         // TODO better support for attributes
         "tests/rust/src/test/run-pass/item-attributes.rs" |
+        // TODO overflows stack unless RUST_MIN_STACK is increased. See comment in file.
+        "tests/rust/src/test/run-pass/super-fast-paren-parsing.rs" |
```

Comment:
Given that [this passes in Travis,](https://travis-ci.org/dtolnay/syn/jobs/223676757#L4570) perhaps I should only exclude it on Windows.